mula/scheduler/context/context.py

```diff
@@ -42,6 +42,14 @@ def __init__(self) -> None:
         with Path(self.config.log_cfg).open("rt", encoding="utf-8") as f:
             logging.config.dictConfig(json.load(f))
 
+        # Set the logging level to the value specified by the env var
+        level = logging.getLevelName(self.config.log_level.upper())
+        root_logger = logging.getLogger()
+        root_logger.setLevel(level)
+        root_logger.handlers[0].setLevel(level)
+        logging.getLogger("uvicorn.access").setLevel(level)
+        logging.getLogger("uvicorn.error").setLevel(level)
```

Comment:
As mentioned offline and maybe for later: we could extend the dict config with this, so we'd have a consistent way of loading the logging configuration and printing out the logging configuration